### DISCUSSION TITLE
Include startpage

### DISCUSSION THREAD

**Discussion by cinnvmonroll** [bromite/bromite#1372] [Ideas]
I like the privacy that duckduckgo provides but I prefer google search results because of their accuracy.

Startpage is the search engine I use because it combines both. So it would be nice if you could add it to the options available in settings to save me from having to go to it's page every time I want to search something.

**Comment by xarantolus** [comment]
You can actually add startpage yourself by following these steps:
* search for something on startpage
* tap the menu at the top right (of the web page, the three lines) and go to their settings
* Scroll to the "HTTP request method" setting and set it to "GET" and save settings
* Now type in any search query in the box on the website and hit enter
* Go to Bromite settings, then search engine 
* Startpage should now be selectable under the "recently visited" search engines

**Reply by cinnvmonroll** [reply]
Thanks a lot.
